src/script/src/table.rs

```diff
@@ -292,6 +291,35 @@ impl ScriptsTable {
     pub fn name(&self) -> &str {
         &self.name
     }
+
+    async fn load_table_rows(
+        table: &TableRef,
+        query_engine: &QueryEngineRef,
+    ) -> Result<SendableRecordBatchStream> {
+        let table_info = table.table_info();
+        let sql = format!("SELECT * FROM {}", table_info.name.as_str());
```

Comment:
Prefer to do this in a structural manner

```suggestion
        let plan = LogicalPlanBuilder::scan_with_filters("t", table, None, vec![])?
            .build();
```

Reply:
so elegant

Reply:
But `TableScan` is unexpected for distributed table.

Reply:
"But TableScan is unexpected for distributed tables, I'm afraid it doesn't work."

Not true, it works